x-pack/test/functional/services/transform/edit_flyout.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export function TransformEditFlyoutProvider({ getService }: FtrProviderContext) {
+  const retry = getService('retry');
+  const testSubjects = getService('testSubjects');
+
+  return {
+    async assertTransfromEditFlyoutExists() {
+      await testSubjects.existOrFail('transformEditFlyout');
+    },
+
+    async assertTransfromEditFlyoutMissing() {
```

Comment:
Typo! Should be `assertTransformEditFlyoutMissing`

Reply:
Fixed in a2613ee.